crates/fiber-lib/src/store/store_impl/mod.rs

```diff
@@ -238,6 +238,33 @@ pub enum KeyValue {
     HoldTlc((Hash256, Hash256, u64), u64),
 }
 
+/// Recorded store changes.
```

Comment:
In long term, we could replace KeyValue with StoreChange. But I did not do this because of two reasons:

- We only need a small portion of store change events to implement multi-hop cch payments
- The KeyValue enum has many nested structs that have not implemented serd traits and serd is required for subscription over RPC.